TileRarities.lua

```diff
@@ -0,0 +1,131 @@
+--[[
+    The data for the different chances and multiplier values can be found on Data:Tile_Chances
+]]--
+local p = {}
+
+function p.main(frame)
+	return p._main(frame.args)
+end
+
+function p._main(args)
+	local json = mw.loadJsonData('Data:Tile_Chances')
+    local ranks = json.ranks
+	local dom = mw.html.create()
+	local tbl = dom:tag('table'):addClass('wikitable'):css('text-align', 'right')
+	local row = tbl:tag('tr')
+	row:tag('th'):attr('rowspan', 2):wikitext('Rarity'):css('vertical-align', 'bottom')
+    for i, v in ipairs(ranks) do
```

Comment:
unindent here, this will be my last comment about indentation but please do not indent unnecessarily, you should indent at logical blocks and only at logical blocks

Reply:
As was already explained on discord is this due to a mix of spaces and tabs and GitHub's different display of those.
In VSCode is it looking consistent.